docs/user/index.rst

```diff
@@ -48,6 +48,7 @@ to help you create fantastic documentation for your project.
   :doc:`With Sphinx </intro/getting-started-with-sphinx>` |
   :doc:`With MkDocs </intro/getting-started-with-mkdocs>` |
   :doc:`/choosing-a-site`
+  :doc:`/examples`
```

Comment:
I think we might want to include this a bit lower down in the docs while we're working to build out this page. I'd be 👍 on shipping it with the current PR if this was perhaps in the `How-to guides` section, and just linked from some of the other pages/guides.

Reply:
:+1: 

* Moved to the How-to guides
* Sprinkled out some cross-referencing